integration_tests/mockApis/mappings/openid-configuration.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This is a copy of the actual [GOV.UK One Login discovery endpoint data](https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/integrate-with-code-flow/#use-the-discovery-endpoint), but with just the hostnames changed.